tests/test_features/test_manifold.py

```diff
@@ -92,16 +92,30 @@ def test_manifold_instance_construction(self):
         oz = Manifold(manifold=manifold)
         assert oz.manifold is manifold
 
-    @patch('yellowbrick.features.manifold.Manifold.draw', spec=True)
-    def test_manifold_fit(self, mock_draw):
+    @patch('yellowbrick.features.manifold.Manifold.fit_transform', spec=True)
+    def test_manifold_fit(self, mock_fit_transform):
         """
         Test manifold fit method
         """
         X, y = make_s_curve(1000, random_state=888)
         manifold = Manifold(target="auto")
 
-        assert not hasattr(manifold, 'fit_time_')
         assert manifold.fit(X, y) is manifold, "fit did not return self"
+        mock_fit_transform.assert_called_once()
+
+    @patch('yellowbrick.features.manifold.Manifold.draw', spec=True)
+    def test_manifold_fit_transform(self, mock_draw):
+        """
+        Test manifold fit_transform method
+        """
+        X, y = make_s_curve(1000, random_state=888)
+        manifold = Manifold(target="auto")
+
+        assert not hasattr(manifold, 'fit_time_')
+
+        Xp = manifold.fit_transform(X, y)
+        assert Xp.shape == (X.shape[0], 2)
+
         mock_draw.assert_called_once()
         assert hasattr(manifold, 'fit_time_')
         assert manifold._target_color_type == CONTINUOUS
```

Comment:
This is the old `test_manifold_fit` test - though now I test to make sure that transformed data is returned with the correct shape.